js/app.js

```diff
@@ -1,2 +1,89 @@
-$(function () {
+$(document).ready(function () {
+  $('#submit-btn').click(() => {
+
+    refreshSearch()
+
+    const city = $('#city-type').val()
+
+    getWeatherByCity(city)
+
+  })
+
+
+  function getWeatherByCity (city) {
+    axios.get(`http://api.openweathermap.org/data/2.5/weather?q=${city}&APPID=1c739a9713a0919a3024a089c3b027df`)
+    .then((response) => {
+      console.log(response.data)
+      displayCityName(response.data.name)
+      displayCurrentTemperature(convertToFahrenheit(response.data.main.temp))
+      displayWeatherDescription(response.data.weather[0].description)
+      displayMinTemp(convertToFahrenheit(response.data.main.temp_min))
+      displayMaxTemp(convertToFahrenheit(response.data.main.temp_max))
+
+    }).catch((error) => {
+      console.log(error)
+    })
+
+    }
+
+
+
+
+  function displayCityName (cityName) {
+    $('#current-city').append(
+      `<h4>Current City:</h4> <p>${cityName}</p>`)
+  }
+
+  function displayCurrentTemperature (temperature) {
+    $('#current-temp').append(
+      `<h4>Current Temperature:</h4> <p>${temperature} F°</p>`)
+      setTempFontColor(temperature)
+  }
+
+  function displayWeatherDescription (description) {
+    $('#description').append(
+      `<h4>Description:</h4> <p>${description}</p>`)
+  }
+
+  function displayMinTemp (temperature) {
+    $('#min-temp').append(
+      `<h4>Today's Low:</h4> <p>${temperature} F°</p>`)
+  }
+
+  function displayMaxTemp (temperature) {
+    $('#max-temp').append(
+      `<h4>Today's High:</h4> <p>${temperature} F°</p>`)
+  }
+
+  function convertToFahrenheit (kelvinUnits) {
+    return  parseInt((kelvinUnits-273.15)*(9/5)+32)
+  }
+
+  function setTempFontColor (temperature) {
```

Comment:
@kcycle great job breaking out this logic into a separate function. 

the only suggestion here would be to use `if..elseif..esle` logic instead of two separate `if` statements.  Take a look at the approach used in the solution: https://github.com/jsd20191008/weather-api-app-solution/blob/master/js/main-promises.js#L82